templates/actions.tmpl.rb

```diff
@@ -117,7 +117,9 @@
 
     output_fields: lambda do |object_definition|
       object_definition['custom_action_output']
-    end
+    end,
+    retry_on_response: [429, 500, 502, 503, 504, 507, 524],
```

Comment:
Cloudlfare also returns a number of other 5XX errors. Can we review those and also support relevant ones?
https://support.cloudflare.com/hc/en-us/articles/115003011431-Troubleshooting-Cloudflare-5XX-errors

Reply:
Thanks @BCook98 
520, 521, 522, 523 looks relevant and can be added. Any other you think makes sense ?